packages/apps/lightning-ui-docs/.storybook/addons/components/ToneRow.js

```diff
@@ -30,7 +30,6 @@ export default function ToneRow({ defaultTone, componentName }) {
       control={
         <OptionsControl
           name="tones"
-          key={`Tones-${componentName}`}
```

Comment:
this isn't needed and should have been removed in the Component Style Panel refactor